operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -676,9 +676,12 @@ function wdtCreatePrimordialDomain() {
 
   if [ "JRF" == "$WDT_DOMAIN_TYPE" ] ; then
     if [ -z "${OPSS_FLAGS}" ] ; then
-      trace "Creating JRF Primordial Domain"
+      trace INFO "An OPSS wallet was not supplied for Model in the Image JRF domain " \
+        "'spec.configuration.opss.walletFileSecret' attribute; therefore, it's assumed that this is the first time " \
+        "the OPSS RCU database is being accessed by the domain, so a schema and a wallet file will be created. " \
+        "Consult the Model in Image documentation for preserving the OPSS wallet file."
```

Comment:
This is the uncorrected suggested contents for the INFO.  Refresh the browser to get the corrected contents.

Reply:
Here are the corrected contents again in case you can't find them easily:

An OPSS wallet was not supplied for the Model in Image JRF domain in its 'spec.configuration.opss.walletFileSecret' attribute; therefore, it's assumed that this is the first time the OPSS RCU database is being accessed by the domain, so a schema and a wallet file will be created. Consult the Model in Image documentation for instructions about preserving the OPSS wallet file.